### PR TITLE
[poco/all]fix:Poco lib add PocoEncodings

### PR DESCRIPTION
**Pull request by daixian**
Specify library name and version:  **poco/all**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

issues：
https://github.com/conan-community/community/issues/325

### CODE REVIEW

File: recipes/poco/all/conanfile.py [hunk 1]
```diff
@@ -190,6 +190,7 @@ def package_info(self):
                 self.cpp_info.libs.append("%s%s" % (lib, suffix))
 
         self.cpp_info.libs.append("PocoFoundation%s" % suffix)
+        self.cpp_info.libs.append("PocoEncodings%s" % suffix)
```
Comment (uilianries) [inline recipes/poco/all/conanfile.py:193]: since when Encodings is mandatory? IIRC only PocoFoundation is available by default. Does it require an option?

Comment (daixian) [reply recipes/poco/all/conanfile.py:193]: emmm... Sorry,I see that this option exists, is it correct to add it here?
``` python
def package_info(self):
    libs = [("enable_mongodb", "PocoMongoDB"),
            ("enable_pdf", "PocoPDF"),
            ("enable_netssl", "PocoNetSSL"),
            ("enable_netssl_win", "PocoNetSSLWin"),
            ("enable_net", "PocoNet"),
            ("enable_crypto", "PocoCrypto"),
            ("enable_data_sqlite", "PocoDataSQLite"),
            ("enable_data_mysql", "PocoDataMySQL"),
            ("enable_data_odbc", "PocoDataODBC"),
            ("enable_data", "PocoData"),
            ("enable_sevenzip", "PocoSevenZip"),
            ("enable_zip", "PocoZip"),
            ("enable_apacheconnector", "PocoApacheConnector"),
            ("enable_util", "PocoUtil"),
            ("enable_xml", "PocoXML"),
            ("enable_json", "PocoJSON"),
            ("enable_redis", "PocoRedis"),
            ("enable_encodings", "PocoEncodings")] #Add PocoEncodings
```

**Comment by danimtb** [comment]
@daixian please solve conflicts

**Comment by daixian** [comment]
It's been fixed

**Pull request closed without merging**